src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml

```diff
@@ -237,9 +243,14 @@
                                 </div>
                                 <div class="tab-pane" id="cadl-help">
                                     <ol>
-                                        <li>
-                                            Provide URL to cadl project root path.
-                                        </li>                                        
+                                        
+                                       Provide URL to cadl project root path.
```

Comment:
The changes in this file are for the description box thing I showed you in the morning, right?

Reply:
correct